app/Http/Controllers/v1/TokenController.php

```diff
@@ -76,7 +78,7 @@ public function create()
                 0,
                 microtime(true),
                 'TokenController Created',
-                TokenToolkit::makeSimpleOTP()
+                $otp
```

Comment:
Nothing going on here, just pre-creating otp, instead of doing it inline, so the variable can be reused in the email and logging output.